CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Unreleased
```

Comment:
Because this is targeting the `master` branch, it'll be released immediately. For that reason, I'd label this section `## v9.10.0`. If this endpoint is useful yet, I'd submit this PR to the `dev` branch instead and not address the changelog yet.